go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/maistra/maistra-test-tool
 
-go 1.15
+go 1.18
```

Comment:
In 2.4 we may use go 1.19 to match the golang version with maistra/istio side. We can change this and dependencies later. This is fine now.

Reply:
+1